tests/rptest/services/redpanda.py

```diff
@@ -234,8 +234,13 @@ def __init__(self,
                  cloud_storage_region="panda-region",
                  cloud_storage_bucket=f"panda-bucket-{uuid.uuid1()}",
                  cloud_storage_api_endpoint="minio-s3",
-                 cloud_storage_api_endpoint_port=9000,
-                 cloud_storage_cache_size=160 * 1000000):
+                 cloud_storage_api_endpoint_port=443,
```

Comment:
Doesn't this break the docker tests that run minio on port 9000?

Reply:
Thanks, I was too eager to push.